staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/interface.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+// GaugeInput is the methods of a gauge that are used by instrumented code.
+type GaugeInput interface {
```

Comment:
I don't like the "Input" word here.

Can we just call it "Gauge" (and similarly for all below stuff)?

[Also, calling it Gauge makes it more related to prometheus Gauge.]

Reply:
Works for me.  I was shy of that because the unadorned word "gauge" in Prometheus means more than just these methods.  @dgrisonnet ?  @logicalhan ?

Reply:
I would say it's fine - it's our own namepace (pkg/util/flowcontrol/metrics).

 @dgrisonnet @logicalhan - if you disagree - please shout :)

Reply:
I also think it is fine to rename it to `Gauge` since it is inside of the flowcontrol metrics namespace.

Reply:
I have done that name simplification.